spec/unit/follow_redirects_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'helper'
 require 'faraday_middleware/response/follow_redirects'
 require 'faraday'
+require 'sinatra/base'
```

Comment:
It would be great to avoid this dependency if possible.
It should be possible to write tests without it and using `Faraday::Adapter::Test` instead, just look at how the other tests have been written.